staging/src/k8s.io/apiserver/pkg/util/webhook/validation.go

```diff
@@ -23,8 +23,18 @@ import (
 
 	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/client-go/transport"
 )
 
+func ValidateCABundle(fldPath *field.Path, caBundle []byte) field.ErrorList {
```

Comment:
add a unit test that ensures:
* empty is valid
* non-empty with a valid cert is valid
* non-empty without any certs is invalid
* non-empty with malformed certs is invalid

Reply:
Added. The case for "non-empty without any certs is invalid" and "non-empty with malformed certs is invalid" is handled in the same way when validating via creating the transport, but added test cases for both.